<!-- -*- mode: XML -*- -->
<!-- run ibeo_had_node for Ibeo Lux/Scala lidar with ECU -->

<launch>

  <!-- declare arguments with default values -->
  <arg name="rate" default="50" />
  <arg name="sub_topic" default="/idc_reader/ibeo_objects" />
  <arg name="text_size" default="1" />
  <arg name="box_size" default="0.2" />

  <!-- node for HAD -->
  <node pkg="ibeo_had" type="ibeo_had_visualizer" name="ibeo_visualizer" output="screen">
    <param name="rate" value="$(arg rate)"/>
    <param name="sub_topic" value="$(arg sub_topic)"/>
    <param name="text_size" value="$(arg text_size)"/>
    <param name="box_size" value="$(arg box_size)"/>
  </node>

</launch>
